### ISSUE TITLE
`FirstMethodArgumentLineBreak` cop doesn't work for `super`

### ISSUE THREAD

**Issue by AlexWayfer** [rubocop-hq/rubocop#6974]
Hello!

I've noticed that `FirstMethodArgumentLineBreak` cop doesn't work for `super` call.

--------

## Expected behavior

Offenses from `FirstMethodArgumentLineBreak` cop about code with `super`.

## Actual behavior

There are no offenses.

## Steps to reproduce the problem

```ruby
super(:foo,
  :bar
)
```

## RuboCop version

Include the output of `rubocop -V` or `bundle exec rubocop -V` if using Bundler. Here's an example:

```
$ [bundle exec] rubocop -V
0.67.2 (using Parser 2.6.2.1, running on ruby 2.5.5 x86_64-linux)
```

**Comment by koic** [comment]
Thanks for your feedback. I opened a PR #6976.

**Comment by AlexWayfer** [comment]
> Thanks for your feedback. I opened a PR #6976.

Thank you for the work and help!
